requirements.txt

```diff
@@ -1,4 +1,4 @@
 featuretools>=0.7.0,<0.12.0
-pandas>=0.23.0,<0.24.0
+pandas>=0.25.0
 python-dateutil>=2.6.1,<2.8.1
 tsfresh>=0.11.2
```

Comment:
we should require tsfresh 0.13.0 or higher since the older versions do not support pandas 0.25

Reply:
updated tsfresh version